workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/README.md

```diff
@@ -0,0 +1,250 @@
+
+1. Install the CI providers you want the plugin to handle the security scan outputs from.
+   The supported CI's are:
+
+- [Jenkins](../../../jenkins/plugins/jenkins-backend/README.md)
+- [Github Actions](../../../github-actions/plugins/github-actions/README.md)
+- [Gitlab CI](https://github.com/immobiliare/backstage-plugin-gitlab?tab=readme-ov-file#setup)
+
+#### Permission Framework Support
+
+The Multi Source Security Viewer (MSSV) plugin has support for the permission framework.
+
+1. When [RBAC permission](https://github.com/backstage/community-plugins/tree/main/workspaces/rbac/plugins/rbac-backend#installation) framework is enabled, for non-admin users to access MSSV UI, the role associated with your user should have the following permission policies associated with it.
+   Add the following in your permission policies configuration file named `rbac-policy.csv`:
+
+```CSV
+p, role:default/team_a, mssv.view.read, read, allow
+```
+
+You can specify the path to this configuration file in your application configuration:
+
+```yaml
+permission:
+  enabled: true
+  rbac:
+    policies-csv-file: /some/path/rbac-policy.csv
+    policyFileReload: true
+```
+
+2. When using the [permission policy](https://backstage.io/docs/permissions/writing-a-policy/) framework.
+
+Configure the backend by [configuring the permission policy](https://github.com/backstage/backstage/blob/master/docs/permissions/getting-started.md#test-permission-policy). You can use below policy as example.
+
+```typescript
+import { createBackendModule } from '@backstage/backend-plugin-api';
+import {
+  PolicyDecision,
+  AuthorizeResult,
+} from '@backstage/plugin-permission-common';
+import {
+  PermissionPolicy,
+  PolicyQuery,
+  PolicyQueryUser,
+} from '@backstage/plugin-permission-node';
+import { policyExtensionPoint } from '@backstage/plugin-permission-node/alpha';
+
+class MssvPermissionPolicy implements PermissionPolicy {
+  async handle(
+    request: PolicyQuery,
+    user: PolicyQueryUser,
+  ): Promise<PolicyDecision> {
+    const isGuestUser = user.info.userEntityRef === 'user:development/guest';
+    if (isGuestUser && request.permission.name === 'mssv.view.read') {
+      return { result: AuthorizeResult.DENY };
+    }
+    return { result: AuthorizeResult.ALLOW };
+  }
+}
+
+export default createBackendModule({
+  pluginId: 'permission',
+  moduleId: 'permission-policy',
+  register(reg) {
+    reg.registerInit({
+      deps: { policy: policyExtensionPoint },
+      async init({ policy }) {
+        policy.setPolicy(new MssvPermissionPolicy());
+      },
+    });
+  },
+});
+```
+
+![mssv-rbac](./docs/images/mssv-rbac.png)
+
+#### Procedure
+
+1. Install the plugin using the following command:
+
+```
+yarn workspace app add @backstage-community/multi-source-security-viewer
+```
+
+To enable the PipelineRun list in the CI/CD tab (or any other tab) on the entity view page, add the following snippet in the packages/app/src/components/catalog/EntityPage.tsx.
```

Comment:
Again, I would not use the CI/CD tab. You're going to have people who have adopted the other plugins already and overloading this tab is not going to be a great UX/DX.

Reply:
Perhaps a Security tab would be best? 🤔